src/server/api/interfaces/filters.ts

```diff
@@ -27,6 +27,7 @@ export const stringFilter = z.object({
   operator: z.enum(filterOperators.string),
   value: z.string(),
   type: z.literal("string"),
+  valueIsRawSQL: z.boolean().optional(),
```

Comment:
@flxwu great contribution! As we expose this via our API for the dashboards, do you think this could allow SQL injection?

Reply:
Great catch!! I believe I even thought about this when I implemented it, must've forgotten to put a note here to revisit this 🤔

Prisma.queryRaw only allows for [1 query at a time](https://www.prisma.io/docs/orm/prisma-client/queries/raw-database-access/raw-queries#sql-injection). 

This of course still allows for SQL injection, but not malicious ones of modifying data or reading undesired tables.

However, just to be more safe & follow best practices, I think it'd be better to disallow this type of RawSQL query from the client API alltogether.

So I'd propose to move this `valueIsRawSQL` filter into a new filter that is only to be used on the server alone.

Wdyt? @maxdeichmann